### PR TITLE
[DOCS-7677] Add Keycloak to ACS 7.4.2 and APS 2.4 SP

### PR DESCRIPTION
**Pull request by mbujak-hyland**
Relates to:

-  #1477 
-  #1478

### CODE REVIEW

File: content-services/7.4/support/index.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Search and Insight Engine 2.0.7 and later | Search and Insight Engine is compatible with Java 11 as long as you run Zeppelin in a Java 8 runtime. You can do this either in a VM or separate Java 8 based server. |
 | Alfresco Search Services 2.0.7 and later | |
 | Alfresco Federation Services 3.0 | |
-| Keycloak 21.1.2 | Use this Keycloak version and Alfresco Keycloak Theme 0.3.5 as an alternative to Identity Service 2.0.0. |
+| Keycloak 24.0.3 | Use this Keycloak version and Alfresco Keycloak Theme 0.3.5 as an alternative to Identity Service 2.0.0. |
```
Comment (anxumalo) [inline content-services/7.4/support/index.md:84]: Looks good to me, cc. @dsibilio - a sanity check please? (_this updates the ACS 7.4.2 table_)

File: process-services/2.4/support/index.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | **Integrations** | |
 | Identity Service 1.2 | For use with LDAP and SAML |
 | Identity Service 1.1 | For use with LDAP and SAML |
+| Keycloak 24.0.3 | Enable `Exclude Issuer from Authentication Response` - see [Configure authentication](https://docs.alfresco.com/process-services/2.4/config/authenticate/#properties){:target="_blank"} for details. |
```
Comment (anxumalo) [inline process-services/2.4/support/index.md:56]: 1. Update the link format for "internal" docs, so it's verified by our build process.
2. Copy the fixed note to the **Process Services 2.4** entry in `content-services/7.4/support/index.md`. _This completes the ACS part of your updates._
3. Agree with Aritri how to complete the rest of the **APS** part of the docs update.

```suggestion
| Keycloak 24.0.3 | Enable `Exclude Issuer from Authentication Response` - see [Configure authentication]({% link process-services/2.4/config/authenticate.md}#properties) for details. |
```

File: content-services/7.4/support/index.md (hunk 1 above)
Comment (dsibilio) [reply content-services/7.4/support/index.md:84]: IMO it's ok, I'm not sure if @vprovaggi prefers to list Keycloak 21.1.2 and Keycloak 24.0.3 separately as potential alternatives to eachother?

File: process-services/2.4/support/index.md (hunk 2 above)
Comment (dsibilio) [reply process-services/2.4/support/index.md:56]: The instructions here are very much more summarized compared to what has been written in the DOCS ticket, I'm not sure it'll be easy for customers to follow through.

cc: @vprovaggi

Comment (mbujak-hyland) [reply process-services/2.4/support/index.md]: The more detailed instructions will be provided in this "Configure authentication" section.

File: content-services/7.4/support/index.md (hunk 1 above)
Comment (dsibilio) [reply content-services/7.4/support/index.md:84]: _(as a personal preference, I'd reckon a single line mentioning both "Keycloak 24.0.3, 21.1.2" would be acceptable in this case)_

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: process-services/2.4/support/index.md [hunk 3]
```diff
@@ -51,8 +51,11 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Elasticsearch 7.17.6 | |
 | | |
 | **Integrations** | |
+| Keycloak 24.0.3 | Enable `Exclude Issuer from Authentication Response` - see [Configure authentication]({% link process-services/2.4/config/authenticate.md %}#properties) for details. |
+| Keycloak 21.1.2 | |
 | Identity Service 1.2 | For use with LDAP and SAML |
 | Identity Service 1.1 | For use with LDAP and SAML |
+
```
Comment (anxumalo) [inline process-services/2.4/support/index.md:58]: Delete this line - it breaks the table

**Pull request merged**